### ISSUE TITLE
Failed acceptance tests

### ISSUE THREAD

**Issue by PaulAgeyev** [adopted-ember-addons/ember-page-title#161]
Hi guys. Thanks for the amazing addon. It works very good.

But, I have found one minor issue with running acceptance tests. 

### Steps to reproduce
Run acceptance tests using the following URL:
> http://localhost:4200/tests?filter=Acceptance:%20title

### Actual results
<img width="1677" alt="Screenshot 2020-06-01 at 10 12 06" src="https://user-images.githubusercontent.com/9153931/83384906-e18fd600-a3f0-11ea-8b45-5b5187d3c7f9.png">

### Root cause
In the `_removeExistingTitleTag` method, the `title` element is removed each time a new test is run.
```
  _removeExistingTitleTag() {
    if (this._hasFastboot()) {
      return;
    }

    let titles = document.getElementsByTagName('title');
    for (let i = 0; i < titles.length; i++) {
      let title = titles[i];
      title.parentNode.removeChild(title);
    }
  },
```

**Comment by knownasilya** [comment]
So the issue is only in the tests but not in the app?

**Comment by PaulAgeyev** [comment]
Yes, you are right.

This issue is reproduced only in the acceptance tests.

**Comment by PaulAgeyev** [comment]
Could you please provide bugfix?

**Comment by knownasilya** [comment]
@PaulAgeyev is this affecting your app? Or your app's tests?

**Comment by knownasilya** [comment]
Fixed with b89ba58, will be included in next release.

**Comment by PaulAgeyev** [comment]
@knownasilya, it's affecting our app's tests..
Thank you!

**Comment by knownasilya** [comment]
The screenshot you showed was for the add-on. What kind of error are you seeing in your app? Are you testing your apps titles?

**Comment by PaulAgeyev** [comment]
Yes, you are right. I wanted to show the root cause.

In our app, we test page titles in acceptance tests, because we have over 100 pages.
We use the same testing approach as you in add-on, but we don't have the dummy data.
Since we do not have dummy data in tests, the tests are failed. We have exactly the same error that you resolved in the add-on's tests.

It looks like this fix should be applied to the add-on's implementation.

**Comment by knownasilya** [comment]
So the issue seems to be how the tests are run, because the issue isn't present when doing `npm test` but is present when using `/tests` UI for the tests. If I fix it for the UI it stops working for the CLI. Trying to get some more insight into the difference.

**Comment by knownasilya** [comment]
@mtoygar does it work with both `npm test` and `/tests` url?

**Comment by mtoygar** [comment]
In my case it is work both for `ember test` and `/tests`. This may be stem from how I use ember-page-title. Below, are my global configs. I do not override below configs in any of my `page-title` helper.

```js
pageTitle: {
      separator: ' ▸ ',
      prepend: false,
      replace: false,
      front: false,
    },
```

The solution may not be valid for some configs, I don't know. Does that makes sense to you @knownasilya ?

**Comment by PaulAgeyev** [comment]
@knownasilya, did you get a chance to look at this?

**Comment by knownasilya** [comment]
@PaulAgeyev atm the workaround @mtoygar is the best bet. There is an issue with the testing setup or the head service that prevents me changing the implementation in the addon. I'll add this to the readme.

**Comment by raido** [comment]
This should be resolved by #168.

**Comment by knownasilya** [comment]
Closing. If issue isn't resolved (test on master before next major is released) please comment here.
